test/extensions/filters/http/wasm/wasm_filter_integration_test.cc

```diff
@@ -12,13 +12,21 @@ namespace {
 
 class WasmFilterIntegrationTest
     : public HttpIntegrationTest,
-      public testing::TestWithParam<std::tuple<std::string, std::string, bool>> {
+      public testing::TestWithParam<
+          std::tuple<std::tuple<std::string, std::string, bool>, Http::CodecType>> {
```

Comment:
optional: you could define a helper function similar to [wasmDualFilterTestMatrix](https://github.com/envoyproxy/envoy/blob/bb28dda78d462851c1810047a4b1488a5a69279a/test/extensions/common/wasm/wasm_runtime.cc#L49-L56) to allow the test to use a flat tuple<std::string, std::string, bool, Http::CodecType> rather than nesting tuples, which gets a little awkward with the nested std::get<>s.